pom.xml

```diff
@@ -545,6 +545,36 @@
           <artifactId>dependency-check-maven</artifactId>
           <version>4.0.2</version>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-checkstyle-plugin</artifactId>
+          <version>3.0.0</version>
+          <dependencies>
+            <dependency>
+              <groupId>com.puppycrawl.tools</groupId>
+              <artifactId>checkstyle</artifactId>
+              <version>8.17</version>
```

Comment:
put this version in <properties> with a name something like checkstyle.plugin.dependency.version ?

Reply:
@nkalmar  done